test/core/security/credentials_test.cc

```diff
@@ -97,6 +97,13 @@ static const char test_refresh_token_str[] =
     "  \"refresh_token\": \"1/Blahblasj424jladJDSGNf-u4Sua3HDA2ngjd42\","
     "  \"type\": \"authorized_user\"}";
 
+static const char test_refresh_token_with_quota_project_id_str[] =
+    "{ \"client_id\": \"32555999999.apps.googleusercontent.com\","
+    "  \"client_secret\": \"EmssLNjJy1332hD4KFsecret\","
+    "  \"refresh_token\": \"1/Blahblasj424jladJDSGNf-u4Sua3HDA2ngjd42\","
+    "  \"quota_project_id\": \"my-quota-project-id\","
```

Comment:
question: quota_project _id could also be a number, right?

Reply:
It is always a string.

"project ID" is always a string
"project number" is always a number

Reply:
When I read the spec, it was fluid. However, is it really a problem if we treat numeric ID as a string?

Reply:
go/adc-spec is a (very) draft spec for the JSON file, and we've specified it as a string.

If in the future we need to support project numbers (and not treat the number as a string), it'll be a separate field, "quota_project_number".